packages/commonwealth/client/scripts/views/components/sidebar/discussion_section.tsx

```diff
@@ -220,6 +225,31 @@ export const DiscussionSection = () => {
     }
   }
 
+  const archivedSectionGroup: SectionGroupAttrs = {
+    title: 'Archived',
+    rightIcon: <CWIcon iconName="archiveTray" iconSize="small" />,
+    containsChildren: false,
+    hasDefaultToggle: false,
+    isVisible: true,
+    isUpdated: true,
+    isActive: !!matchesArchivedRoute,
+    onClick: (e, toggle: boolean) => {
+      e.preventDefault();
+      handleRedirectClicks(
+        navigate,
+        e,
+        `/archived`,
+        app.activeChainId(),
+        () => {
+          setDiscussionsToggleTree(`children.Archived.toggledState`, toggle);
+        }
+      );
+    },
+    displayData: null,
+  }
```

Comment:
prettier: file is not formatted correctly

Reply:
Done!